deps/error_codes.jl

```diff
@@ -35,9 +35,9 @@ function generate_error_codes(io, html=error_code_html())
     id_names = Set{String}()
 
     class_enum_io = IOBuffer()
-    println(class_enum_io, "@cenum(Class,")
+    println(class_enum_io, "@cenum(\n    Class,")
     error_enum_io = IOBuffer()
-    println(error_enum_io, "@cenum(ErrorCode,")
+    println(error_enum_io, "@cenum(\n    ErrorCode,")
```

Comment:
This is why I wanted to split out this formatting change; I think we need to indicate in the output that this is the name of the enum, but I'm not sure how to do that. 

Maybe:

```julia
@cenum(
    # enum type
    ErrorCode
    # enum values
    ...
```
?